compiler/rustc_driver_impl/src/lib.rs

```diff
@@ -237,6 +237,9 @@ fn run_compiler(
         Box<dyn FnOnce(&config::Options) -> Box<dyn CodegenBackend> + Send>,
     >,
 ) -> interface::Result<()> {
+    // Throw away the first argument, the name of the binary
+    let at_args = at_args.split_first().map_or(at_args, |(_, args)| args);
```

Comment:
nitpick: this can be written a bit simpler:
```suggestion
    let at_args = at_args.get(1..).unwrap_or(at_args);
```

Reply:
Oh, right.